src/Components/TruckCard/TruckCard.js

```diff
@@ -75,6 +77,8 @@ const handleDownvote = () => {
     setIsDownvoted(!isDownvoted)
     if (!isDownvoted) {
       setDownVote(downVote += 1)
+      // props.truck.up_rating += -1 also parse int if backend data is a string 
+      // add line of code to direclty manipulate the data model
       props.truck.userDownvote = true
     } else {
       setDownVote(downVote += -1)
```

Comment:
The upvote and downvote are so similar, that I wonder if there is some way to make only one function that is dynamic for both features?